### PR TITLE
Develop a TODO app

### PR DESCRIPTION
**Pull request by robinbenitezmora**
It was completed the next tasks in this PR:
[Part 1 - React tutorial: The beginner's guide to learning React in 2020](https://ibaslogic.com/react-tutorial-for-beginners/).
[Part 2 - Working with React form and handling event](https://ibaslogic.com/react-form-handling/).
Create a new repository work on a new branch (use Gitflow)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,81 @@
...
+Then, move to the newly created folder `to-do-react-app` using this command
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+`cd to-do-react-app`
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+Then, install dependencies using this command
 
-## Learn More
+Then, install dependencies using this command
```
Comment (carlosmuerto) [inline README.md:56]: check this description in your README.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,81 @@
...
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- React
+- JavaScript
+- HTML
+- CSS
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## 💻 Live demo
+- [Live Demo Link](https://github.com/robinbenitezmora/to-do-react-app)
```
Comment (carlosmuerto) [inline README.md:17]: If you going to implement gh-pages, don't forget fix this link

File: src/index.js [hunk 3]
```diff
@@ -1,25 +1,12 @@
...
-// // If you want to start measuring performance in your app, pass a function
-// // to log results (for example: reportWebVitals(console.log))
-// // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-// reportWebVitals();
+ReactDOM.render(
+  <React.StrictMode>
+    <TodoContainer />
+  </React.StrictMode>,
+  document.getElementById('root'),
+);
```
Comment (carlosmuerto) [inline src/index.js:12]: This form of Init the rendering is deprecated use what comes default in the create-react-app instead.
```
Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot 
```

File: src/reportWebVitals.js [hunk 4]
```diff
@@ -1,4 +1,5 @@
-const reportWebVitals = onPerfEntry => {
+/*eslint-disable*/
+const reportWebVitals = (onPerfEntry) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
     import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
       getCLS(onPerfEntry);
```
Comment (carlosmuerto) [inline src/reportWebVitals.js:5]: This Component looks like it's not used, please remove it if not used.

File: src/App.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+/*eslint-disable*/
```
Comment (carlosmuerto) [inline src/App.js:1]: This Component looks like it's not used, please remove it if not used.

**Review by carlosmuerto — CHANGES_REQUESTED** [review]
# Request Changes
Good job! this looks promising. there is some change I wanna address.

## general comments
- [ ] please don't disable Linters at the file level, Linters are important for code readability.
- [ ] Please Address in the PR description a resume of what kind of change are you implementing.
- [ ] you are using git-flow, but your beach name doesn't follow the conversion `feature/<added-feature-name>` in this case could be `feature/todo`

### please check in-line comments for more in-depth details.

File: README.md (hunk 1 above)
Comment (robinbenitezmora) [reply README.md:56]: Thanks for helping Carlos. I have done some changes, especially correcting linters errors, however, functionality is not over. Please check my changes and give me your opinion.

**Review by carlosmuerto — APPROVED** [review]
Hi @robinbenitezmora , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

 - [Part 1 - React tutorial: The beginner's guide to learning React in 2020](https://ibaslogic.com/react-tutorial-for-beginners/). ✅ 
 - [Part 2 - Working with React form and handling event](https://ibaslogic.com/react-form-handling/). ✅ 
- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅

**Pull request merged**